apps/question-bank/build.gradle

```diff
@@ -2,13 +2,30 @@ import org.springframework.boot.gradle.plugin.SpringBootPlugin
 
 plugins {
     id 'jvm-test-suite'
-
+    id 'jacoco'
     alias libs.plugins.springBoot
     alias libs.plugins.springDependencies
 }
 
 group = 'gov.cdc'
 version = '0.0.1-SNAPSHOT'
+// Displays cucumber test results
+apply from: file('../../scripts/gradle/cucumber-logging.gradle')
```

Comment:
We should be able to apply this to all sub-projects from the root.  It might end up displaying the test results for each module.